corehq/apps/cloudcare/templates/formplayer/session_list.html

```diff
@@ -16,6 +16,14 @@
 </script>
 
 <script type="text/template" id="session-view-list-web-apps-template">
+    <div class="module-banner alert alert-info">
+        <span>
+            {% blocktrans %}
+                Web Apps deletes any Incomplete Forms that are more than 7 days old.
```

Comment:
should not this come with trans tag

Reply:
blocktrans should be doing the same thing, as I understand it

Reply:
Yaa correct. overlooked that. All good